sdk/tests/conformance2/rendering/blitframebuffer-size-overflow.html

```diff
@@ -70,8 +70,8 @@
     gl.bindTexture(gl.TEXTURE_2D, tex1);
     gl.texImage2D(gl.TEXTURE_2D, 0, gl.RGBA8, width, height, 0, gl.RGBA, gl.UNSIGNED_BYTE, null);
     var fb1 = gl.createFramebuffer();
-    gl.bindFramebuffer(gl.READ_FRAMEBUFFER, fb1);
-    gl.framebufferTexture2D(gl.READ_FRAMEBUFFER, gl.COLOR_ATTACHMENT0, gl.TEXTURE_2D, tex1, 0);
```

Comment:
The code snippet between line 62 and line 67 has been attached a texture to read fbo. So the code here tend to attach a texture to draw fbo.